common/dynamicconfig/constants.go

```diff
@@ -4502,6 +4504,12 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "EnableStrongIdempotencySanityCheck enables sanity check for strong idempotency",
 		DefaultValue: false,
 	},
+	EnableRoundRobinLoadBalancer: DynamicBool{
```

Comment:
Since we will potentially experiment with different load balancing strategies, let's avoid bool flags and define a string (enum) property indicating maching load balancer strategy to use.